jsonnet/dashboard.jsonnet

```diff
@@ -18,3 +18,19 @@ local etcdMixin = (import 'github.com/etcd-io/etcd/contrib/mixin/mixin.libsonnet
     'etcd.json': std.manifestJsonEx(etcdMixin.grafanaDashboards['etcd.json'], '    '),
   },
 }
+{
+  apiVersion: 'v1',
+  kind: 'ConfigMap',
+  metadata: {
+    annotations: {
+      'include.release.openshift.io/self-managed-high-availability': 'true',
+      'include.release.openshift.io/single-node-developer': 'true',
+      'release.openshift.io/delete': "true",
+    },
+    labels: {
+      'console.openshift.io/dashboard': 'true',
+    },
+    name: 'remove-etcd-dashboard',
```

Comment:
This should be `grafana-dashboard-etcd` (the name of the resource you want to delete)